command/build_test.go

```diff
@@ -20,6 +20,17 @@ import (
 	shell_local "github.com/hashicorp/packer/provisioner/shell-local"
 )
 
+var (
+	spaghettiCarbonara = `spaghetti
+carbonara
+`
+	lasagna = `lasagna
+tomato
+mozza
+cooking...
+`
```

Comment:
Yeah, its not an extremely precise cooking list lol.

Reply:
You say precise, I say pasta ;-)

Reply:
:D